extensions/sendbird/v1/actions/updateTicket/updateTicket.test.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  mockedChannelNames,
+  mockedTicketData,
+  SendbirdClientMockImplementation,
+} from '../../client/__mocks__'
+import { generateTestPayload } from '../../../../../src/tests'
+import { updateTicket } from './updateTicket'
+
+jest.mock('../../client')
+
+describe('Update ticket', () => {
+  const onComplete = jest.fn()
+  const onError = jest.fn()
+
+  const basePayload = generateTestPayload({
+    pathway: {
+      id: 'pathway-id',
+      definition_id: 'pathway-definition-id',
+    },
+    activity: {
+      id: 'activity-id',
+    },
+    patient: { id: 'test-patient' },
+    fields: {
+      ticketId: mockedTicketData.id,
+      relatedChannelUrls: `${mockedChannelNames.channel1},${mockedChannelNames.channel2}`,
+      priority: mockedTicketData.priority,
+    },
+    settings: {
+      applicationId: 'applicationId',
+      chatApiToken: 'chatApiToken',
+      deskApiToken: 'deskApiToken',
+    },
+  })
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('Should call the onComplete callback', async () => {
+    await updateTicket.onActivityCreated(basePayload, onComplete, onError)
+
+    expect(
+      SendbirdClientMockImplementation.deskApi.updateTicket
+    ).toHaveBeenCalledWith(mockedTicketData.id, {
+      relatedChannelUrls: `${mockedChannelNames.channel1},${mockedChannelNames.channel2}`,
+      priority: mockedTicketData.priority,
+    })
+    expect(onComplete).toHaveBeenCalledTimes(1)
+    expect(onError).not.toHaveBeenCalled()
+  })
```

Comment:
Can you expand the tests to include more scenarios please

Reply:
I've added a new one. I can't think of any more tests to add to this code.